spec/controllers/idv/review_controller_spec.rb

```diff
@@ -348,15 +348,10 @@ def show
         end
 
         it 'creates an `account_verified` event once per confirmation' do
-          expect(Event).to receive(:create).with(
-            hash_including(
-              user_id: user.id,
-              event_type: :account_verified,
-              ip: '0.0.0.0',
-            ),
-          )
-
           put :create, params: { user: { password: ControllerHelper::VALID_PASSWORD } }
+          disavowal_event_count = user.events.where(event_type: :account_verified, ip: '0.0.0.0').
+            where.not(disavowal_token_fingerprint: nil).count
+          expect(disavowal_event_count).to eq 1
```

Comment:
if we want to get fancy with Rspec...

```suggestion
          expect { put :create, params: { user: { password: ControllerHelper::VALID_PASSWORD } } }.
            to(change do
              user.events.where(event_type: :account_verified, ip: '0.0.0.0').
                where.not(disavowal_token_fingerprint: nil).
                count
            end.from(0).to(1))
```

Reply:
yeah `.by(1)` is also a good one to use